<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBeaker</class>
 <widget class="QDialog" name="CreateBeaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="tbName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblManufacturer">
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="tbManufacturer"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbEnabled">
     <property name="text">
      <string>Enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateBeaker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateBeaker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
